cpu/numactl.py

```diff
@@ -38,6 +38,9 @@ def setUp(self):
         Source:
         https://github.com/numactl/numactl
         '''
+        self.add_warn = False
+        if len(memory.numa_nodes_with_memory()) < 2:
```

Comment:
Can this check be done just before the `self.log.warn()` ?

Reply:
We can, but is it a suggestion to have a local variable rather than a `self` attribute? Otherwise it is more aligned to be at `setUp` phase.

Reply:
Yes, local check is suggested, and variable's use can also be removed.

Reply:
sure, will make the change.